src/test-utils-new/extractor.ts

```diff
@@ -35,14 +39,21 @@ export default function extractDocumentation(sourceFile: ts.SourceFile, checker:
   const definitions: Array<TestUtilsDoc> = [];
 
   for (const symbol of exportSymbols) {
+    const className = symbol.getName();
+    if (extraExports.includes(className)) {
+      continue;
+    }
     if (!(symbol.flags & ts.SymbolFlags.Class)) {
       throw new Error(`Exported symbol is not a class, got ${checker.symbolToString(symbol)}`);
     }
-    const className = symbol.getName();
+
     const classType = checker.getTypeAtLocation(extractDeclaration(symbol));
     const classDefinition: TestUtilsDoc = { name: className, methods: [] };
     for (const property of classType.getProperties()) {
-      const declaration = extractDeclaration(property);
+      const declaration = property.valueDeclaration;
+      if (!declaration) {
+        throw new Error(`Unexpected member on ${className} – ${property.getName()}`);
```

Comment:
This line is not covered in tests. Because typescript says `valueDeclaration` could be undefined, but it always exists in our cases